conduit/fair/data/datamodules/tabular/acs.py

```diff
@@ -0,0 +1,43 @@
+from dataclasses import dataclass, field
+from typing_extensions import override
+
+from conduit.data import TrainValTestSplit
+from conduit.data.datamodules.tabular import CdtTabularDataModule
+from conduit.fair.data.datasets.acs import (
+    ACSDataset,
+    ACSHorizon,
+    ACSSetting,
+    ACSState,
+    ACSSurvey,
+    ACSSurveyYear,
+)
+from conduit.transforms import TabularNormalize, ZScoreNormalize
+
+__all__ = ["ACSDataModule"]
+
+
+@dataclass(kw_only=True)
+class ACSDataModule(CdtTabularDataModule[ACSDataset]):
+    setting: ACSSetting
+    survey_year: ACSSurveyYear = ACSSurveyYear.YEAR_2018
+    horizon: ACSHorizon = ACSHorizon.ONE_YEAR
+    survey: ACSSurvey = ACSSurvey.PERSON
+    states: list[ACSState] = field(default_factory=lambda: [ACSState.AL])
+
+    @override
+    def _get_splits(self) -> TrainValTestSplit[ACSDataset]:
+        data = ACSDataset(
+            setting=self.setting,
+            survey_year=self.survey_year,
+            horizon=self.horizon,
+            survey=self.survey,
+            states=self.states,
+        )
+        val_data, test_data, train_data = data.random_split(
```

Comment:
Didn't we want to split conditionally?

Reply:
Hmm, you mean that hyaline and support-matching will want to do their own splitting, based on W&B artifacts? If so, then I agree it's a problem.

Reply:
You probably meant something else by "split conditionally" but I don't know what.